### PR TITLE
Got frontend server running

### PR DESCRIPTION
**Pull request by JohnMealy23**
This PR will cause `npm run dev` to start the frontend server.

Note that if the server fails to start, the issue is probably in your code.  You'll see an output of errors in the terminal.  Fix these to get the FE server to start.

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script type="module" src="./src/index.tsx"></script>
+    </head>
+    <body>
+        <div id="root"></div>
```
Comment (JohnMealy23) [inline frontend/index.html:7]: Pop quiz: Why did I give this div the ID of `root`?

Comment (JohnMealy23) [reply frontend/index.html:7]: (If you need a hint, search the repo for "root")

Comment (Huntsr98) [reply frontend/index.html:7]: we use it for getElementById, but I don't remember it's use.  I tried looking in the document we use, but it has a ton of 'root' references...

**Review by Huntsr98 — APPROVED** [review]
looks good!

File: frontend/index.html (hunk 1 above)
Comment (JohnMealy23) [reply frontend/index.html:7]: @Huntsr98 - Sounds like you found the line in the JS where this comes in.  Can you decipher from that line the significance of this div?

**Pull request merged**